website/webpack.config.js

```diff
@@ -26,15 +26,6 @@ const plugins = [
   // so that code patch will never be executed.
   new webpack.IgnorePlugin(/^eslint$/, /babel-eslint/),
 
-  // Prettier //
-
-  // typescript-eslint-parser is a dev dependency of prettier, so it's not
-  // installed by default but prettier does still require it. See
-  // https://github.com/prettier/prettier/issues/986
-  new webpack.IgnorePlugin(/typescript-eslint-parser/, /\/prettier/),
```

Comment:
If you want to upgrade Prettier, you need to provide new regexes to exclude irrelevant parsers (they're structured differently in Prettier now). Unless you want to allow all of the parsers, but then need to check that transform actually works with each of them.